api/turing/imagebuilder/imagebuilder.go

```diff
@@ -176,20 +181,17 @@ func (ib *imageBuilder) waitForJobToFinish(job *apibatchv1.Job) error {
 	for {
 		select {
 		case <-timeout:
-			log.Errorf("timeout waiting for kaniko job completion %s", job.Name)
 			return ErrTimeoutBuildingImage
 		case <-ticker.C:
 			j, err := ib.clusterController.GetJob(ib.imageConfig.BuildNamespace, job.Name)
 			if err != nil {
-				log.Errorf("unable to get job status for job %s: %v", job.Name, err)
```

Comment:
Why was it removed?

Reply:
Ah this should be back there, it was noisy when i was dealing with some race condition.